packages/uniforms-antd/src/BaseForm.tsx

```diff
@@ -1,10 +1,20 @@
 import { BaseForm } from 'uniforms';
+import classnames from 'classnames';
```

Comment:
Keep imports ordered. I won't add a comment for each re-ordering, just not to spam the comments.